src/main/webapp/app/sharing/sharing.component.html

```
@@ -0,0 +1,121 @@
+<body>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)